webpack.common.config.js

```diff
@@ -34,6 +34,8 @@ var entry = {
     'files-page': staticPath('js/pages/files-page.js'),
     'profile-settings-page': staticPath('js/pages/profile-settings-page.js'),
     'profile-account-settings-page': staticPath('js/pages/profile-account-settings-page.js'),
+    'profile-settings-applications-list-page': staticPath('js/pages/profile-settings-applications-list.js'),
```

Comment:
Minor, but the filenames should reflect the entry point names, i.e. add `-page` to the filenames.